raiden/tests/integration/test_send_queued_messages.py

```diff
@@ -88,53 +93,49 @@ def run_test_send_queued_messages(raiden_network, deposit, token_addresses, netw
         routing_mode=RoutingMode.PRIVATE,
     )
 
-    app0.stop()
-    del app0  # from here on the app0_restart should be used
-    app1.start()
-
+    del app0
     app0_restart.start()
 
-    waiting.wait_for_healthy(app0_restart.raiden, app1.raiden.address, network_wait)
-    waiting.wait_for_healthy(app1.raiden, app0_restart.raiden.address, network_wait)
+    # XXX: There is no synchronization among the app and the test, so it is
```

Comment:
If you think the commented-out check is important, please turn it into a TODO comment before merging.

Reply:
I think the commented out check is a confirmation that the task is there... this is implicitly checked by the balance asserts at the end of the test (the task was created and the transfer went through completely).

Reply:
The commented out code is flaky, the comment explains it. I left it there with a comment to tell people that this should not be done, leaving the code is easier than writing a description of the buggy code that doesn't work.